ot/bregman.py

```diff
@@ -612,7 +639,7 @@ def sinkhorn_log(a, b, M, reg, numItermax=1000,
         if log:
             log = {'err': []}
 
-        Mr = M / (-reg)
+        Mr = - M / reg
```

Comment:
Keep the original, it is more efficient
![image](https://user-images.githubusercontent.com/1386873/139801965-ac485c0a-2a34-442b-a4ce-f36b0c9d23af.png)